src/webrtc.ts

```diff
@@ -89,13 +89,13 @@ export class WebrtcExtension extends EventEmitter {
 
   configureStanzaJingle () {
     Object.assign(this.client._stanzaio.jingle.config.peerConnectionConfig, {
-      sdpSemantics: 'unified-plan',
-      iceTransportPolicy: this.config.iceTransportPolicy || 'all'
```

Comment:
the problem here is iceTransportPolicy isn't set at bootstrap and this is the only place this is set.